platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/config/StateConfig.java

```diff
@@ -60,6 +60,7 @@
  * @param suspiciousSignedStateAge      The age of a signed state which is considered to be suspicious. Suspicious
  *                                      states cause a large amount of data to be logged that helps to debug the
  *                                      potential state leak.
+ * @param signedStateAgeCheckPeriod     The period of time between checks for signed states that are suspiciously old.
```

Comment:
I don't think this is precisely correct- the check itself occurs at the configured rate of the `SignedStateSentinal` heartbeat

This value is the minimum period between *notifications* of suspiciously old signed states. Maybe a better name would be `signedStateAgeNotificationRateLimit` (a mouthful, admittedly).

I would also be fine with leaving the name as-is, but clarifying in the doc what this truly represents

Reply:
modified to match the direction of your suggestions.